include/swift/Runtime/Win32.h

```diff
@@ -0,0 +1,55 @@
+//===--- Win32.h - Win32 utility functions ----------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Utility functions that are specific to the Windows port.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_RUNTIME_WIN32_H
```

Comment:
It might be worth moving `_swift_win32_withDbgHelpLibrary()` into this file as well. (Note that it _is_ necessary to continue to export that symbol because DbgHelp is a mess.)

Reply:
I've moved that. It does mean that `Win32.h` will now bring in `windows.h`, as we need `HANDLE` for that.